### PR TITLE
Chore/update dwc libs

### PR DESCRIPTION
**Pull request by monik182**

### CODE REVIEW

File: javascript/webapp/src/hooks/loadProject.ts [hunk 1]
```diff
@@ -275,15 +306,39 @@ function preProcessTables(
...
+				table: table?.table as ColumnTable,
+				name: table.name,
+			}
+			inputs.set(table.name, tableContainer)
+		})
+		const [workflow] = wf
+		// setWorkflow && setWorkflow(workflow)
+		const graph = createGraphManager(inputs, workflow)
+		const outputName = [...graph.outputs].pop() || ''
+		return graph.latest(outputName)
```
Comment (monik182) [inline javascript/webapp/src/hooks/loadProject.ts:341]: @darthtrevino, Here I tried to replace the `runPipelineFromProjectFiles` with the `createGraphManager` method

File: javascript/webapp/src/hooks/loadProject.ts [hunk 2]
```diff
@@ -223,16 +238,32 @@ function preProcessTables(
...
+					id: table.name,
+					table: fetched[index] as ColumnTable,
+					name: table.name,
+				}
+				inputs.set(table.name, tableContainer)
+			})
+
+			const graph = createGraphManager(inputs, stepPostLoad)
+			const outputName = [...graph.outputs].pop() || ''
+			const resultPipeline = graph.latest(outputName)
```
Comment (monik182) [inline javascript/webapp/src/hooks/loadProject.ts:259]: @darthtrevino, Here I was trying to do the same as in `processDataTables`

File: javascript/webapp/public/data/examples/smoking-cessation.json [hunk 3]
```diff
@@ -329,146 +329,209 @@
...
+                        "column": "smoke_intensity_change_pct",
+                        "criteria": [
+                            {
+                                "operator": "=",
+                                "type": "value",
+                                "value": "-1"
+                            }
+                        ]
+                    },
+                    "verb": "binarize",
```
Comment (darthtrevino) [inline javascript/webapp/public/data/examples/smoking-cessation.json:509]: Is this node being used?

File: javascript/webapp/public/data/examples/smoking-cessation.json [hunk 4]
```diff
@@ -329,146 +329,209 @@
...
+                    "args": {
+                        "to": "subject_inclusion",
+                        "column": "weight_change",
+                        "criteria": [
+                            {
+                                "operator": "is not empty"
+                            }
+                        ]
+                    },
+                    "verb": "binarize",
```
Comment (darthtrevino) [inline javascript/webapp/public/data/examples/smoking-cessation.json:527]: Is this node being used?

File: README.md [hunk 5]
```diff
@@ -36,7 +36,7 @@ In other words, ShowWhy enables emulation of randomized controlled trials that p
 
 ### Windows
 
-Open a Poweshell terminal, [navigate to the directory](https://docs.microsoft.com/en-us/powershell/scripting/samples/managing-current-location?view=powershell-7.2) where this project is located and run one of the commands at the `Commands` section of this README.
+Open a Powershell terminal, [navigate to the directory](https://docs.microsoft.com/en-us/powershell/scripting/samples/managing-current-location?view=powershell-7.2) where this project is located and run one of the commands at the `Commands` section of this README.
```
Comment (darthtrevino) [inline README.md:39]: `PowerShell` is probably the right casing

**Review by darthtrevino — APPROVED** [review]
It looks like there may be some cruft in the JSON specs, but it looks good otherwise

**Pull request merged**